contracts/contracts/sbtc-bootstrap-signers.clar

```diff
@@ -20,11 +17,43 @@
 ;;
 
 ;; public functions
-;;
+;; Rotate keys
+;; Used to rotate the keys of the signers. This is called whenever
+;; the signer set is updated.
+(define-public (rotate-keys-wrapper (new-keys (list 15 (buff 32))) (multi-sig-address principal) (new-aggregate-pubkey (buff 32)))
```

Comment:
I think in the Clarity WG call we mentioned that it's better to construct the multisig principal prove the public keys, instead of accepting a param. That way, we can be 100% sure that the keys and the multisig match.

Also totally fine to add a TODO here and open a new issue for that related code and tests

Reply:
Copy. Will open a TODO but then I'm confused on whether the keys will ever actually be stored in the contract? Is your thinking that signer keys should NOT be stored here?? Only a multi-sig principal?

Reply:
I think both, as the signers will use the keys as their "source of truth", but just that the multi-sig will be computed instead of taken as an argument